prism-agent/service/server/src/main/scala/io/iohk/atala/castor/controller/DIDEndpoints.scala

```diff
@@ -90,11 +91,12 @@ object DIDEndpoints {
 
   // MUST conform to https://w3c-ccg.github.io/did-resolution/#bindings-https
   val getDID: PublicEndpoint[
-    String,
+    (RequestContext, String),
     Nothing,
     DIDResolutionResult,
     Any
   ] = infallibleEndpoint.get
+    .in(extractFromRequest[RequestContext](RequestContext.apply))
```

Comment:
@patlo-iog you did this endpoint right?
Was there any particular reason why it doesn't have a context?

Reply:
This one is DID resolution. endpoint  It is a public endpoint with no specific wallet context (assuming you're referring to `WalletAccessContext`).

Reply:
I'm referring to RequestContext. Basically I need the headers of the request

Reply:
Ah got it. No particular reason to not add it. LGTM